protocol/x/clob/abci_test.go

```diff
@@ -66,28 +62,6 @@ func assertFillAmountAndPruneState(
 		exists, _, _ := k.GetOrderFillAmount(ctx, orderId)
 		require.False(t, exists)
 	}
-
-	for blockHeight, orderIds := range expectedPruneableBlockHeights {
```

Comment:
Removing this chunk of code because `expectedPruneableBlockHeights` was never set, so this doesn't ever test anything.